pkg/proxy/endpoints.go

```diff
@@ -349,8 +349,8 @@ func getLastChangeTriggerTime(annotations map[string]string) time.Time {
 	}
 	val, err := time.Parse(time.RFC3339Nano, annotations[v1.EndpointsLastChangeTriggerTime])
 	if err != nil {
-		klog.Warningf("Error while parsing EndpointsLastChangeTriggerTimeAnnotation: '%s'. Error is %v",
-			annotations[v1.EndpointsLastChangeTriggerTime], err)
+		klog.InfoS("Error while parsing EndpointsLastChangeTriggerTimeAnnotation", "error", err,
```

Comment:
`ErrorS(err, ...`

Reply:
@thockin here https://github.com/kubernetes/kubernetes/pull/104891#discussion_r709232121 @aojea suggested that InfoS should be use (previously I was using ErrorS), and I think @aojea is also correct, but yes, ErrorS can also be used.
Please guide me which one should I use here, thank you!

Reply:
My rationale is that error messages use to bring more admin attention and people use to report them.  However, in a case like this, with an endpoint with an invalid annotation for the `EndpointsLastChangeTriggerTimeAnnotation` , this will be generating error messages in the logs but they will be harmless, just noise

Reply:
Anyway, I defer to Tim ... :)

Reply:
/cc thockin please guide

Reply:
Well, this will still log as Info(), so it's not less noisy.  This really does indicate that SOMETHING made a mistake, right?  They are not harmless - they are used for monitoring.

I say ErrorS